manifests/php.yml

```diff
@@ -275,6 +275,8 @@ tests/:
       Test_Debugger_Probe_Snaphots: irrelevant
     test_debugger_probe_status.py:
       Test_Debugger_Probe_Statuses: irrelevant
+    test_debugger_symdb.py:
+      Test_Debugger_SymDb: irrelevant
```

Comment:
Why is this irrelevant and not missing feature?

Reply:
Hmmm, you're right, I aligned with other definitions for this library.
I will fix it in separate PR.